matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/IMXGroupEncryption.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
```

Comment:
Now that it's defined in the app, this is not the correct Copyright (I know this is not nice to manage multiple Copyrights)

Reply:
Oh, good catch 👍

Reply:
this on is still sdk, no?

Reply:
Oh, Benoit marked the wrong class but I fixed the correct one (OutboundSessionKeySharingStrategy)

Reply:
Yes, sorry :)